### ISSUE TITLE
About the learning rate schedule

### ISSUE THREAD

**Issue by Cyyyyyb** [fordevoted/UtilityIR#6]
Hello Author,

I am very interested in your research and have reproduced your code. However, I have some questions regarding the learning rate schedule. Could you please clarify how the learning rate was set during the first 40 epochs and the subsequent 30 epochs?

Thank you very much for your work and your time!

**Comment by fordevoted** [comment]
Thank you for your interest. The learning rate will be set to 1e-4 for the main training stage and 1e-5 for the fine-tuning stage.
